src/CarbonAware.LocationSources/src/LocationSource.cs

```diff
@@ -2,6 +2,7 @@
 using CarbonAware.Interfaces;
 using CarbonAware.Model;
 using CarbonAware.LocationSources.Configuration;
+using CarbonAware.LocationSources.Model;
```

Comment:
May be worth including a comment in the general PR explaining why we're changing the `LocationSources` project and what changes we had to make